var/spack/repos/builtin/packages/pasta/package.py

```diff
@@ -38,7 +39,5 @@ def install(self, spec, prefix):
         install_tree(".", prefix)
```

Comment:
Can you try removing the `install` method and using `PythonPackage`? Would like to change this for the same reasons as busco, we don't want to install all these READMEs, docs, tests, etc.

Reply:
I'll try, or at the very least make the same changes we had with busco.